.gitmodules

```diff
@@ -809,3 +809,6 @@
 [submodule "vendor/grammars/reason"]
 	path = vendor/grammars/reason
 	url = https://github.com/facebook/reason
+[submodule "vendor/grammars/sublime-mzn"]
+	path = vendor/grammars/sublime-mzn
+	url = https://github.com/astenmark/sublime-mzn
```

Comment:
The Atom one is based on the sublime one and they give identical output for all the tests I ran, so I don't know if there's a good criterion for deciding between them.